protocol/messenger_communities.go

```diff
@@ -924,11 +924,13 @@ func (m *Messenger) RequestToJoinCommunity(request *requests.RequestToJoinCommun
 		return nil, err
 	}
 
-	adminsAndTokenMasters := community.GetPrivilegedMembers()
-	for _, communityAdmin := range adminsAndTokenMasters {
-		_, err := m.sender.SendPrivate(context.Background(), communityAdmin, &rawMessage)
-		if err != nil {
-			return nil, err
```

Comment:
What if someone will modify his client and send you such a message?
I think we need additionally to filter out messages on client side (HandleRequestToJoin)

Reply:
Yes, need to drop the message as discussed